stock_release_channel_shipment_advice_process_end_time/models/stock_release_channel.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import timedelta
+
+from odoo import api, fields, models
+
+
+class StockReleaseChannel(models.Model):
+
+    _inherit = "stock.release.channel"
+    process_end_time_delay = fields.Integer(
```

Comment:
As for the previous comment, this name doesn't reflect its usage... It should be renamed and a help text would be appreciated.